### PR TITLE
CIV-13686 application summary

### PR DESCRIPTION
**Pull request by GorilaNaranja**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-13686

### Change description ###
Application Summary page - GA for LiPs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by david-jeronimo — DISMISSED** [review]

File: src/main/routes/features/dashboard/claimantDashboardController.ts [hunk 1]
```diff
@@ -99,10 +99,7 @@ const getSupportLinks = (claim: Claim, claimId: string, lng: string) => {
     claim.ccdState === CaseState.DECISION_OUTCOME;
 
   const showGetDebtRespiteLink = claim.ccdState === CaseState.AWAITING_RESPONDENT_ACKNOWLEDGEMENT ||
-    claim.ccdState === CaseState.AWAITING_APPLICANT_INTENTION ||
```
Comment (neeta-hmcts) [inline src/main/routes/features/dashboard/claimantDashboardController.ts:102]: keep the changes for other states

File: src/main/routes/features/dashboard/claimantDashboardController.ts [hunk 2]
```diff
@@ -113,7 +110,7 @@ const getSupportLinks = (claim: Claim, claimId: string, lng: string) => {
   if (showTellUsEndedLink) {
     iWantToLinks.push({ text: t('PAGES.DASHBOARD.SUPPORT_LINKS.TELL_US_ENDED', { lng }), url: constructResponseUrlWithIdParams(claimId, DATE_PAID_URL) });
   }
-  if (showGetDebtRespiteLink && claim.isClaimant()) {
```
Comment (neeta-hmcts) [inline src/main/routes/features/dashboard/claimantDashboardController.ts:116]: same here.. this link suppose to be visible only to claimant

File: src/main/routes/features/dashboard/claimantDashboardController.ts (hunk 1 above)
Comment (GorilaNaranja) [reply src/main/routes/features/dashboard/claimantDashboardController.ts]: deleted changes

File: src/main/routes/features/dashboard/claimantDashboardController.ts (hunk 2 above)
Comment (GorilaNaranja) [reply src/main/routes/features/dashboard/claimantDashboardController.ts]: deleted changes

**Review by neeta-hmcts — APPROVED** [review]

**Review by david-jeronimo — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

**Review by manish14836 — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**